_posts/2017/2017-05-09-how-to-successfully-run-a-company-tech-blog.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "How to successfully run a company tech blog"
+date: "2017-05-09 10:00:00"
+image: blog-header/social-like.jpg
+categories: tech-stories
+authors: ["Rose"]
+---
+
+At [ePages](https://www.epages.com/us/) we develop e-commerce solutions. Thus, we had enough techies around that supported us setting up a [company tech blog](https://developer.epages.com/blog).
+Our [API documentation](https://developer.epages.com/apps) was already [Jekyll](https://jekyllrb.com/)-based. As Jekyll is a blog engine, it came natural to use it also as a platform for our tech blog.
```

Comment:
In Jekyll vielleicht? Irgendwie hört es sich komisch an so ganz ohne was davor:D